src/Drivers/Database.php

```diff
@@ -23,10 +23,26 @@ public function audit(Auditable $model): ?Audit
     public function prune(Auditable $model): bool
     {
         if (($threshold = $model->getAuditThreshold()) > 0) {
-            return $model->audits()
+            $class = get_class($model->audits()->getModel());
+            $auditKeyName = (new $class)->getKeyName();
+
+            $forRemoval = $model->audits()
                 ->latest()
-                ->offset($threshold)->limit(PHP_INT_MAX)
-                ->delete() > 0;
+                ->get()
+                ->slice($threshold)
+                ->pluck($auditKeyName);
```

Comment:
It seems to me that it is more efficient to put the `pluck` above the `slice`, replacing the `get`